src/Sunburst.ts

```diff
@@ -442,6 +712,10 @@ export class Sunburst implements IVisual {
         this.renderTooltip(pathSelectionMerged);
         this.setCategoryLabelPosition(this.viewport.width);
         this.setPercentageLabelPosition(this.viewport.width);
+        this.main.selectAll("path." + this.appCssConstants.slice.className)
+            .attr("role","option")
+            .attr("tabindex","0");
+        this.main.selectAll("text").attr("role","presentation");
```

Comment:
move `.attr("role", "presentation")` near to `.append("textPath")` because it is expensive operation.

Reply:
add aria-label lke this: `.attr('aria-label', (d: HierarchyRectangularNode<SunburstDataPoint>) => d.data.name);` 

Screen reader will announce value for list items.

Reply:
Done: [57cc2e5](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/57cc2e5bf79b2464761d36e369c7fb0138cc4c54)